### PR TITLE
Add SadisticMystic as a researcher

### PR DESCRIPTION
**Pull request by DaWoblefet**

### CODE REVIEW

**Comment by Zarel** [comment]
@SadisticMystic do you want to be credited by name?

File: README.md [hunk 1]
```diff
@@ -183,6 +183,7 @@ Contributors
 - Leonard Craft III [DaWoblefet] - Research (game mechanics)
 - Luke Harmon-Vellotti [moo, CheeseMuffin] - Development
 - Robin Vandenbrande [Quinella] - Development
+- [SadisticMystic] - Research (game mechanics)
```
Comment (Zarel) [inline README.md]:
```suggestion
- Russell Jones [SadisticMystic] - Research (game mechanics)
```

Hah, it's still in the right place alphabetically, too.

**Comment by SadisticMystic** [comment]
That's fine.

**Comment by AustinXII** [comment]
Hm is there any reason the credits for the server repo and the client repo are different? It seems that the server credits isn’t what is used in the credit section on the actual server. It may just need a sync if that’s how that works idk

**Comment by Zarel** [comment]
What credits in the client repo?

I'm pretty sure there are no credits in the client repo.

If you mean the credits on the client website, yeah, those need to be synced.

**Comment by AustinXII** [comment]
@Zarel https://github.com/Zarel/Pokemon-Showdown-Client/blob/e5a31785976b0a665bc60316aa58be76b6e07ad6/js/client-mainmenu.js#L1095 this specifically is out of date on the client.

**Comment by Zarel** [comment]
@AustinXII You're linking to a commit from 2017, of course it's out of date.

81befa2760db5c1a212 is also out of date, because it's from 2017. :|

**Comment by AustinXII** [comment]
@Zarel oh I didn’t realize that sorry i’m on mobile, I just remembered there being credits on client at one point and I got that after searching the repo. Sorry I should have searched more thoroughly it just seemed like that may have still been being used elsewhere on the client so I thought I’d still bring it up. Thanks for the info on the syncing though :)

**Pull request merged**